#62 Fixed first screen

PawelMelnik opened the pull request:
- Surface view changed on onDraw
- Fixed incorrect image after rotation
- Fixed losing parametrs of the grid
- Added dialog if not entered sizes
- Fixed Seekbar saving

PawelMelnik committed:
#62 Fixed first screen

- Surface view changed on onDraw
- Fixed incorrect image after rotation
- Fixed losing parametrs of the grid
- Added dialog if not entered sizes
- Fixed Seekbar saving

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentConvertDialog.java:
```diff
@@ -0,0 +1,39 @@
+
+public class FragmentConvertDialog extends DialogFragment{
+
+    public Dialog onCreateDialog(Bundle savedInstanceState) {
```
Please annotate this method as `@Override`

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentConvertDialog.java:
```diff
@@ -0,0 +1,39 @@
+
+    public Dialog onCreateDialog(Bundle savedInstanceState) {
+        FragmentActivity fragmentActivity = getActivity();
+        if (fragmentActivity == null) return null;
```
This method annotated with `@NonNull` return `null` (check documentation)

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentConvertDialog.java:
```diff
@@ -0,0 +1,39 @@
+    public Dialog onCreateDialog(Bundle savedInstanceState) {
+        FragmentActivity fragmentActivity = getActivity();
+        if (fragmentActivity == null) return null;
+        androidx.appcompat.app.AlertDialog.Builder builder =
```
Import `androidx.appcompat.app.AlertDialog` and use `AlertDialog.Builder`.

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentConvertDialog.java:
```diff
@@ -0,0 +1,39 @@
+                .setIcon(R.drawable.ic_info)
+                .setMessage(R.string.msg_convert_ok)
+                .setPositiveButton(R.string.action_ok, listenerOk);
+        return builder.create();
```
It can be replaced by
```java
return new AlertDialog.Builder(getActivity())
                .setTitle(R.string.title_dialog_convert)
                .setIcon(R.drawable.ic_info)
                .setMessage(R.string.msg_convert_ok)
                .setPositiveButton(R.string.action_ok, listenerOk)
                .create();
```

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentConvertDialog.java:
```diff
@@ -0,0 +1,39 @@
+        }
+    };
+
+    public void onDismiss(DialogInterface dialog) {
```
Please annotate this method as `@Override`

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentConvertDialog.java:
```diff
@@ -0,0 +1,39 @@
+        super.onDismiss(dialog);
+    }
+
+    public void onCancel(DialogInterface dialog) {
```
Please annotate this method as `@Override`

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentConvertDialog.java:
```diff
@@ -0,0 +1,39 @@
+    };
+
+    public void onDismiss(DialogInterface dialog) {
+        super.onDismiss(dialog);
```
Do we need this method? Our method `onDismiss` can be deleted.

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentConvertDialog.java:
```diff
@@ -0,0 +1,39 @@
+    }
+
+    public void onCancel(DialogInterface dialog) {
+        super.onCancel(dialog);
```
Do we need this method? Our method `onCancel` can be deleted.

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentMain.java:
```diff
@@ -26,20 +25,20 @@ public void onCreate(Bundle savedInstanceState) {
+        View viewMainFragment = inflater.inflate(R.layout.fragment_main, null);
 
-        _viewMainFragment
+        viewMainFragment
```
Please check [Method chain case](https://github.com/nonoteam/NonoConverter/blob/master/CodingStandard.md#line-wrapping-strategies) in CodingStandard.md

PawelMelnik replied:
I should group that code into the one chain?
`View viewMainFragment = inflater.inflate(R.layout.fragment_main,null);
        viewMainFragment.findViewById(R.id.button_crop)
                .setOnClickListener((View.OnClickListener) getActivity());
        viewMainFragment.findViewById(R.id.button_columns)
                .setOnClickListener((View.OnClickListener) getActivity());`

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentMain.java:
```diff
@@ -26,20 +25,20 @@ public void onCreate(Bundle savedInstanceState) {
                 .findViewById(R.id.button_crop)
                 .setOnClickListener((View.OnClickListener) getActivity());
-        _viewMainFragment
+        viewMainFragment
```
Please check [Method chain case](https://github.com/nonoteam/NonoConverter/blob/master/CodingStandard.md#line-wrapping-strategies) in CodingStandard.md

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentMain.java:
```diff
@@ -26,20 +25,20 @@ public void onCreate(Bundle savedInstanceState) {
-        ImageView ivMain = (ImageView)_viewMainFragment.findViewById(R.id.image_view_main);
-        ivMain.setImageBitmap(editImageActivity._bmpCurrentImage);
+        ImageView ivMain = viewMainFragment.findViewById(R.id.image_view_main);
+        if (editImageActivity != null) ivMain.setImageBitmap(editImageActivity._bmpCurrentImage);
```
This line has a length over than 80

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentConvertDialog.java:
```diff
@@ -0,0 +1,39 @@
+import androidx.fragment.app.DialogFragment;
+import androidx.fragment.app.FragmentActivity;
+
+public class FragmentConvertDialog extends DialogFragment{
```
It looks like `package-private`

PawelMelnik replied:
Android studion said that it must be public here

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/Panel.java:
```diff
@@ -58,78 +56,67 @@
     }
 
     /**
      * Draw grid over the image
```
Please change this comment or delete this method and use method `invalidate`

maik-nack commented on app/src/main/res/values/strings.xml:
```diff
@@ -23,6 +23,9 @@
     <string name="title_dialog_done">Done</string>
     <string name="action_ok">OK</string>
+    <string name="title_dialog_convert">Nonogram size</string>
+    <string name="msg_convert_ok">Conversion can not be started.\nPlease click on "Columns"
```
This line has a length over than 80

maik-nack replied:
If you want to use `" "` you should be writing `\"Columns\"`

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/EditImageActivity.java:
```diff
@@ -77,24 +107,31 @@
+     * Flag for starting convert
      */
-    static final String _fragmentColumnsTag = "_fragmentColumns";
+    boolean _isSelectedColumns = false;
```
Do we need this variable?

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/EditImageActivity.java:
```diff
@@ -115,58 +152,78 @@ protected void onCreate(Bundle savedInstanceState) {
+                        .findFragmentByTag(_FRAGMENT_MAIN_TAG);
+        if (_fragmentMain != null && _fragmentMain.getView()!= null) {
+            ((ImageView)_fragmentMain
+                    .getView()
```
Please check [Method chain case](https://github.com/nonoteam/NonoConverter/blob/master/CodingStandard.md#line-wrapping-strategies) in CodingStandard.md

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/EditImageActivity.java:
```diff
@@ -115,58 +152,78 @@ protected void onCreate(Bundle savedInstanceState) {
+                    .getInt(_IS_SELECTED_COLUMNS_TAG, 0) != 0;
+            byte[] byteArray = savedInstanceState.getByteArray(_BMP_CURRENT_IMAGE_TAG);
+            if (byteArray != null)
+                _bmpCurrentImage =
```
Please check [`if`](https://github.com/nonoteam/NonoConverter/blob/master/CodingStandard.md#use-standard-brace-style) in CodingStandard.md

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/EditImageActivity.java:
```diff
@@ -115,58 +152,78 @@ protected void onCreate(Bundle savedInstanceState) {
+        ByteArrayOutputStream stream = new ByteArrayOutputStream();
+        _bmpCurrentImage.compress(Bitmap.CompressFormat.PNG, 100, stream);
+        byte[] byteArray = stream.toByteArray();
+        outState.putByteArray(_BMP_CURRENT_IMAGE_TAG, byteArray);
```
Try to put `String` - a path of the image

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/EditImageActivity.java:
```diff
@@ -115,58 +152,78 @@ protected void onCreate(Bundle savedInstanceState) {
+        outState.putInt(_COUNT_COLUMNS_TAG, _columns);
+        outState.putInt(_COUNT_ROWS_TAG, _rows);
+
+        if (_isSelectedColumns) {
```
You can try
```java
        outState.putByte(_IS_SELECTED_COLUMNS_TAG, 
                _isSelectedColumns ? (byte) 1 : (byte) 0);
```

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/EditImageActivity.java:
```diff
@@ -115,58 +152,78 @@ protected void onCreate(Bundle savedInstanceState) {
+        if (savedInstanceState != null) {
+            _columns = savedInstanceState.getInt(_COUNT_COLUMNS_TAG, 0);
+            _rows = savedInstanceState.getInt(_COUNT_ROWS_TAG, 0);
+            _isSelectedColumns = savedInstanceState
```
You can try
```java
            _isSelectedColumns = savedInstanceState
                    .getByte(_IS_SELECTED_COLUMNS_TAG, (byte) 0) != 0;
```

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/EditImageActivity.java:
```diff
@@ -115,58 +152,78 @@ protected void onCreate(Bundle savedInstanceState) {
+            _rows = savedInstanceState.getInt(_COUNT_ROWS_TAG, 0);
+            _isSelectedColumns = savedInstanceState
+                    .getInt(_IS_SELECTED_COLUMNS_TAG, 0) != 0;
+            byte[] byteArray = savedInstanceState.getByteArray(_BMP_CURRENT_IMAGE_TAG);
```
This line has a length over than 80

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/EditImageActivity.java:
```diff
@@ -115,58 +152,78 @@ protected void onCreate(Bundle savedInstanceState) {
+            byte[] byteArray = savedInstanceState.getByteArray(_BMP_CURRENT_IMAGE_TAG);
+            if (byteArray != null)
+                _bmpCurrentImage =
+                        BitmapFactory.decodeByteArray(byteArray, 0, byteArray.length);
```
This line has a length over than 80

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/EditImageActivity.java:
```diff
@@ -115,58 +152,78 @@ protected void onCreate(Bundle savedInstanceState) {
             _fragmentMain = new FragmentMain();
             _fragmentTransaction = getSupportFragmentManager().beginTransaction();
-            _fragmentTransaction.add(R.id.fragment_layout_edit, _fragmentMain, _fragmentMainTag);
+            _fragmentTransaction.add(R.id.fragment_layout_edit, _fragmentMain, _FRAGMENT_MAIN_TAG);
```
This line has a length over than 80

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/EditImageActivity.java:
```diff
@@ -177,14 +234,14 @@ public boolean onOptionsItemSelected(MenuItem item) {
-
-                getSupportFragmentManager().popBackStack();
+                if (!_isSelectedColumns) {
+                    fragmentConvertDialog.show(getSupportFragmentManager(), _DIALOG_CONVERT_TAG);
```
This line has a length over than 80

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/EditImageActivity.java:
```diff
@@ -277,11 +305,56 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
+                _fragmentTransaction.replace(R.id.fragment_layout_edit, _fragmentMain);
+                break;
+            case 2:
+                _fragmentTransaction =  getSupportFragmentManager().beginTransaction();
```
This line has a length over than 80

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -108,149 +154,198 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case R.id.menu_done:
+                EditImageActivity editImageActivity = (EditImageActivity)getActivity();
```
This line has a length over than 80

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -108,149 +154,198 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+                if (editImageActivity != null) {
+                    editImageActivity.setSizes(_countRows, _countColumns);
+                    editImageActivity._isSelectedColumns = true;
+                    editImageActivity.getSupportFragmentManager().popBackStack();
```
This line has a length over than 80

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -108,149 +154,198 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
+        boolean isSaved = false;
         _viewColumnsFragment = inflater.inflate(R.layout.fragment_columns, null);
```
This line has a length over than 80

maik-nack replied:
Compare with `FragmentMain.java`

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -108,149 +154,198 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         _viewColumnsFragment = inflater.inflate(R.layout.fragment_columns, null);
-        boolean is_saved = false;
         _panel = _viewColumnsFragment.findViewById(R.id.panel);
         _ivColumns = _viewColumnsFragment.findViewById(R.id.image_view_columns);
```
This line has a length over than 80

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -108,149 +154,198 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
-
         EditImageActivity editImageActivity = (EditImageActivity)getActivity();
-        _bitmap = editImageActivity._bmpCurrentImage;
+        if (editImageActivity != null) _bitmap = editImageActivity._bmpCurrentImage;
```
This line has a length over than 80

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -108,149 +154,198 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
                     .setText(
-                            String.valueOf(_count_columns) + "," + String.valueOf(_count_rows));
-            is_saved = true;
+                            String.format("%s%s%s", String.valueOf(_countColumns), _COMMA, String.valueOf(_countRows)));
```
This line has a length over than 80

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -108,149 +154,198 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+    }
+
+    @Override
+    public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
```
This line has a length over than 80

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -108,149 +154,198 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+                    (_imageViewHeight - _height)/2;
+            _startWidth =
+                    0;
+        } else {
```
Why there are so many line breaks in this function?

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -108,149 +154,198 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+    private void setSizes(boolean is_saved) {
         Context context = getContext();
+        if (context == null) return;
         WindowManager wm = (WindowManager) context.getSystemService(Context.WINDOW_SERVICE);
```
This line has a length over than 80

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -108,149 +154,198 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
 
         int resourceId = getResources()
-                .getIdentifier("status_bar_height", "dimen", "android");
+                .getIdentifier(_IDENTIFIER_NAME, _IDENTIFIER_DEF_TYPE, _IDENTIFIER_DEF_PACKAGE);
```
This line has a length over than 80

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -108,149 +154,198 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         int resourceId = getResources()
-                .getIdentifier("status_bar_height", "dimen", "android");
+                .getIdentifier(_IDENTIFIER_NAME, _IDENTIFIER_DEF_TYPE, _IDENTIFIER_DEF_PACKAGE);
         int status_bar_height = getResources().getDimensionPixelSize(resourceId);
```
This line has a length over than 80

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -108,149 +154,198 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+        if (!_isLandscape){
+            _imageViewWidth = screenWidth;
+            _imageViewHeight =
+                    screenHeight - tb_height - status_bar_height - (int)(60 * metrics.density);
```
This line has a length over than 80

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -108,149 +154,198 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+        int screenWidth = size.x;
+        int screenHeight = size.y;
 
         TypedArray styledAttributes = getContext().getTheme().obtainStyledAttributes(
```
This line has a length over than 80. Please check [Method chain case](https://github.com/nonoteam/NonoConverter/blob/master/CodingStandard.md#line-wrapping-strategies) in CodingStandard.md

maik-nack replied:
Please check

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -108,149 +154,198 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
      * @param is_saved true if SavedInstance != null
      */
-    public void computeSizes(boolean is_saved) {
+    private void setSizes(boolean is_saved) {
```
Name of variable

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -108,149 +154,198 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
-        DisplayMetrics metrics = new DisplayMetrics();
-        display.getMetrics(metrics);
+        if (!is_saved) {
+            EditImageActivity editImageActivity = (EditImageActivity)getActivity();
```
This line has a length over than 80

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -108,149 +154,198 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
-        display.getMetrics(metrics);
+        if (!is_saved) {
+            EditImageActivity editImageActivity = (EditImageActivity)getActivity();
+            if (editImageActivity != null) _countColumns = editImageActivity._columns;
```
This line has a length over than 80

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -108,149 +154,198 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
-            _start_h = 0;
+            _countRows = (int)(_countColumns * _height * 1.0 / (_width * 1.0));
+            _tvRowsAndColumns
+                    .setText(String.format("%s%s%s", String.valueOf(_countColumns), _COMMA, String.valueOf(_countRows)));
```
This line has a length over than 80

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -108,149 +154,198 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
-    @Override
-    public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
-
+        _panel.setLengths(_startWidth, _startHeight, _width, _height, _countRows, _countColumns);
```
This line has a length over than 80

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -108,149 +154,198 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
-        Panel panel = _viewColumnsFragment.findViewById(R.id.panel);
-        panel.doDraw(_start_w, _start_h, _width, _height, _count_rows, _count_columns);
+        _countRows = (int)(_countColumns * _height * 1.0 / (_width * 1.0));
+        _panel.setLengths(_startWidth, _startHeight, _width, _height, _countRows, _countColumns);
```
This line has a length over than 80

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentMain.java:
```diff
@@ -23,20 +23,22 @@ public void onCreate(Bundle savedInstanceState) {
         ImageView ivMain = viewMainFragment.findViewById(R.id.image_view_main);
-        if (editImageActivity != null) ivMain.setImageBitmap(editImageActivity._bmpCurrentImage);
+        if (editImageActivity != null)
+            ivMain.setImageBitmap(editImageActivity._bmpCurrentImage);
```
Where are `{ }`?

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/EditImageActivity.java:
```diff
@@ -335,19 +338,20 @@ void setSizes(int rows, int columns) {
      */
     public void changeFragment(int count) {
         switch (count) {
             case 1:
```
What means these numbers? Is it possible to make their constants?

maik-nack replied:
Look at line 271

maik-nack commented:
Please check `Class member ordering` in [Coding standard](http://github.com/nonoteam/NonoConverter/blob/master/CodingStandard.md#class-member-ordering) in `EditImageActivity.java`

maik-nack commented:
Create private method in `FragmentColumns.java` for
```java
            _tvRowsAndColumns.setText(String.format("%s%s%s",
                    String.valueOf(_countColumns),
                    COMMA,
                    String.valueOf(_countRows)));
```

PawelMelnik committed:
#62 Fixed first screen

- Fixed incorrect coding standard
- Fixed columns and rows output
- Fixed changing orientation

PawelMelnik committed:
#62 Fixed first screen

- Fixed coding standard mistakes
- Added constants to switch in EditImageActivity

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -180,8 +180,8 @@ public void onSaveInstanceState(Bundle outState) {
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
```
Compare with `FragmentMain.java`

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -305,8 +305,7 @@ private void setSizes(boolean isSaved) {
-                getTheme()
+        TypedArray styledAttributes = getContext().getTheme()
                 .obtainStyledAttributes(
                 new int[] { android.R.attr.actionBarSize });
```
It needs 8 spaces

PawelMelnik replied:
It have 8 spaces

PawelMelnik replied:
Sorry, looked another line

PawelMelnik committed:
#62 Fixed first screen

- Fixed coding standard
- Changed setting text in FrarmentColumns

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -236,12 +235,16 @@ public void onStartTrackingTouch(SeekBar seekBar) {}
+        _panel.invalidate();
+    }
+
+    private void setTextViewRowsAndColumnsText() {
```
Where is a comment?

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -108,149 +154,211 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+                .getIdentifier(IDENTIFIER_NAME,
+                        IDENTIFIER_DEF_TYPE,
+                        IDENTIFIER_DEF_PACKAGE);
+        int status_bar_height =
```
Name of variable

maik-nack commented on app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java:
```diff
@@ -108,149 +154,211 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+        TypedArray styledAttributes = getContext().getTheme()
+                .obtainStyledAttributes(
+                        new int[] { android.R.attr.actionBarSize });
+        int tb_height =
```
Name of variable

PawelMelnik committed:
#62 Fixed first screen

- Changed variable names
- Added comments

maik-nack reviewed (approved):
Good job!
